src/shared/sgxlkl_enclave_config_gen.c

```diff
@@ -0,0 +1,178 @@
+/* Automatically generated from ../../tools/schemas/enclave-config.schema.json; do not modify. */
```

Comment:
If it's auto-generated, why check it in?

Reply:
Same reason as for the other file! Just makes life easier.